R/table.R

```diff
@@ -63,31 +63,49 @@ html_table.xml_node <- function(x, header = NA, trim = TRUE,
 
   ncols <- lapply(cells, html_attr, "colspan", default = "1")
   ncols <- lapply(ncols, as.integer)
+  nrows <- lapply(cells, html_attr, "rowspan", default = "1")
+  nrows <- lapply(nrows, as.integer)
```

Comment:
Overall, this PR speaks to, but does not completely solve #111. I left a comment on that issue basically saying that the same approach, i.e. filling in with replication, could be taken with colspan. The only issue is then in the first row where the resulting data.frame would potentially have to try to interpret multiple columns with the same name. The simple solution would be that R just be allowed to interpret as usual, in the sense that `names(data.frame(X=c(1,2,3), X=c(2,3,4)))` is simply  `c('X', 'X.1')`. Let me know if you're happy to take the same approach with colspan and I'll make that change part of the same PR.